source/includes/templates/tabs-gs.rst

```diff
@@ -16,17 +16,17 @@
    .. raw:: html
 
       <div class="tabs">
-        <ul class="nav nav-tabs nav-justified">
+        <ul tab-strip tab-strip--singleton" role="tablist">
           {{ for tab in tabs getLanguageNames}}
           {{ # Only render the tab here if i < 5 }}
           {{ if i lessThan(5) }}
-          <li{{ if i zero }} class="active"{{ end }}><a href="#{{ tab.id }}">{{ tab.name }}</a></li>
+          <li class="tab-strip__element" data-tabid="{{ tab.id }}" role="tab" aria-selected="{{ if i zero }}true{{ else }}false{{ end }}">{{ tab.name }}</li>
```

Comment:
Can we use a more descriptive name like "label" or something since "element" is part of the name "block__element--modifier"? If so, reflect changes below.